src/AppBundle/Subscriber/BugLabelNewIssueSubscriber.php

```diff
@@ -36,15 +36,15 @@ public function onIssues(GitHubEvent $event)
         }
 
         $responseData = array('issue' => $issueNumber = $data['issue']['number']);
-        // Ignore non-bugs or issue which already has a status
+        // Ignore non-bugs or issue which already has a statu
         if ('bug' !== strtolower($data['label']['name']) || null !== $this->statusApi->getIssueStatus($issueNumber, $repository)) {
             $responseData['status_change'] = null;
             $event->setResponseData($responseData);
 
             return;
         }
 
-        $newStatus = Status::NEEDS_REVIEW;
+        $newStatus = Status::UNCONFIRMED;
```

Comment:
In case of a bug confirmed does not mean we have a valid solution for it and it may need some other reviews, on the contrary it might stay unconfirmed and set reviewed so we can close it.